index.html

```diff
@@ -6,18 +6,54 @@
     <title>Awesome Books</title>
     <link rel="stylesheet" href="style/index.css" />
   </head>
+  
   <body>
-    <h1>Awesome Books</h1>
-    <div id="bookList"></div>
+    <header>
+      <div class= "nav">
+     <a href="#" class="navbar-logo">Awesome Books</a>
+      <nav class="navbar">
+        <ul class="nav-list">
+          <li class="navbar-list bookList">List</li>
+          <li class="navbar-list form">Add new</li>
+          <li class="navbar-list contact">Contact</li>
+        </ul>
+       </nav>
```

Comment:
nice work so far 👍🏿  in your project but as per project requirements when any of the links are clicked, the main area changes to display only one of the three content sections. you can make use of clssList in css

Reply:
This change has been done well